source/documentation/administration_guide/topics/Introduction_to_Hosts.adoc

```diff
@@ -13,7 +13,7 @@ KVM can concurrently host multiple virtual machines running either Windows or Li
 You can identify the host type of an individual host in the {virt-product-fullname} {engine-name} by selecting the host's name. This opens the details view. Then look at the *OS Description* under *Software*.
 ====
 
-Hosts use `tuned` profiles, which provide virtualization optimizations. For more information on `tuned`, see the link:{URL_rhel_docs_legacy}html/Performance_Tuning_Guide/sect-Red_Hat_Enterprise_Linux-Performance_Tuning_Guide-Performance_Monitoring_Tools-tuned_and_tuned_adm.html[_{enterprise-linux} 7 Performance Tuning Guide_].
+Hosts use `tuned` profiles, which provide virtualization optimizations. For more information on `tuned`, see the link:{URL_rhel_docs_legacy}html/Performance_Tuning_Guide/sect-Red_Hat_Enterprise_Linux-Performance_Tuning_Guide-Performance_Monitoring_Tools-tuned_and_tuned_adm.html[Red Hat Enterprise Linux 7 Performance Tuning Guide_].
```

Comment:
Missing an underscore before title. You must check changes like this with a local build of the downstream doc.